docs/configuration.md

```diff
@@ -45,7 +45,7 @@ For optimal performance, it is recommended to enable the data model acceleration
 |--------|--------|-------------|
```

Comment:
Lets remove macro configuration from Cyences Settings page.
Remove macro column from the doc here.